src/cli/commands/login.command.ts

```diff
@@ -38,14 +40,21 @@ export class LoginCommand {
             return Response.badRequest('Email address is invalid.');
         }
 
-        if ((password == null || password === '') && canInteract) {
-            const answer: inquirer.Answers = await inquirer.createPromptModule({ output: process.stderr })({
-                type: 'password',
-                name: 'password',
-                message: 'Master password:',
-            });
-            password = answer.password;
+        if (password == null || password === '') {
+            if (cmd.passwordFile) {
+                password = await NodeUtils.readFirstLine(cmd.passwordFile);
+            } else if (cmd.passwordEnv && process.env[cmd.passwordEnv]) {
```

Comment:
Can you change these to `passwordenv` and `passwordfile`. Looking at our other multi-word CLI options, everything is lowercase. Otherewise, looks good to go.

Reply:
Done. Will mean that these are --paswordfile and --passwordenv instead of --password-file and --password-env in the command then.